src/TBuffer.cpp

```diff
@@ -2472,6 +2472,7 @@ inline int TBuffer::wrap(int startLine)
     QStringList timeList;
     QList<bool> promptList;
     int lineCount = 0;
+    TChar pSpace(mpConsole);
```

Comment:
As this does not change through the iterations of the `for` loop it should not be recreated each time!